website/docs/getting-started/setup/installation-guides/docker/README.mdx

```diff
@@ -106,7 +108,9 @@ Automatic updates are turned off by default in the docker-compose.yml file. To e
 
 :::info
 To complement automatic updates, Appsmith also has an auto backup feature. Before an automatic update, Appsmith first creates a backup to allow [rolling back](/getting-started/setup/instance-management/appsmithctl#restore-appsmith-instance) an update to a previous version of Appsmith if necessary. For this auto backup to work, please ensure that there is a minimum of 2 GB of free storage; otherwise, both the backup and update tasks fail.
-:::
+Appsmith relies on the Watchtower hook to perform an auto-backup before an update. Please ensure the WATCHTOWER_LIFECYCLE_HOOKS env is enabled on the Watchtower instance. Refer to the docker-compose.yaml at https://bit.ly/3h08WqB.
+
+Please refer to this(/getting-started/setup/instance-management/appsmithctl#restore-appsmith-instance) to rollback the Appsmith data using the backup archives.
```

Comment:
```suggestion
If you update your Appsmith instance and face any issues you can rollback the changes and [restore the Appsmith instance](/getting-started/setup/instance-management/appsmithctl#restore-appsmith-instance) from a backup archive.
```

keep rollback as one work although Grammarly suggests otherwise as users tend to type it as one word